azavea/sentinel-2-stac-utils
# Include metadata files

## Overview

This PR adds the productInfo.json asset for every item produced in L1C and L2A catalogs. 

### Checklist

- ~New tests have been added or existing tests have been modified~ lol someday :(

### Notes

L1C doesn't have a pre-baked cloud mask, so the L2A pre-baked cloud mask is used for both collections.

### Testing Instructions

- Create a catalog for L1C and L2A:
  - [ ] `bloop run s2stac -- create --collection L1C --inventory-path some-bogus-value --output-catalog-root test/with/assets/l1c`
  - [ ] `bloop run s2stac -- create --collection L1C --inventory-path some-bogus-value --output-catalog-root test/with/assets/l2a`
- check an item for each:
  - [ ] open an item from `test/with/assets/l1c/48/X/WG/2019/08/21/items/`
  - [ ] make sure its `productInfo` asset is an actually existing s3 path (you'll need `--request-payer requester` in the aws cli command)
  - [ ] open an item from `test/with/assets/l2a/6/U/UG/2019/01/10/items/`
  - [ ] make sure its `productInfo` asset is an actually existing s3 path (you'll need `--request-payer requester` in the aws cli command)
- confirm that items have `xml-metadata`, `product-info` and `tile-info` assets

Closes #9
